src/styles/_catalog.scss

```
@@ -1,22 +1,34 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(4, $cardWidth);
+  gap: 48px;
+  justify-content: center;
 
-  max-width: 944px;
-  margin: 0 auto;
+  @media (max-width: 1024px) {
+    & {
+      grid-template-columns: repeat(3, $cardWidth);
+    }
```

```suggestion
      grid-template-columns: repeat(3, $cardWidth);
```

`@meida` is applied to the block it is defined in 😉

---

Not done.